source/documentation/deploying_services/opensearch.md

```diff
@@ -176,6 +176,32 @@ cf delete-service my-opensearch-service
 
 Enter `yes` when asked for confirmation.
 
+<h2 id="migrating-from-elasticsearch">Migrating from Elasticsearch</h2>
+
+GOV.UK PaaS will remove support for Elasticsearch in the first quarter of 2022. All existing Elasticsearch services will need to be migrated to OpenSearch.
+
+You can migrate from Elasitcsearch to OpenSearch by creating an OpenSearch backing service from a backup of an Elasticsearch service.
+
+```
+cf create-service opensearch OPENSEARCH_PLAN OPENSEARCH_SERVICE_NAME -c '{"restore_from_latest_backup_of": "ELASTICSEARCH_SERVICE_NAME"}'
+```
+
+where:
+
+- `OPENSEARCH_PLAN` is the name of the OpenSearch service plan you want
+- `OPENSEARCH_SERVICE_NAME` is a unique descriptive name for the new service instance
+- `ELASTICSEARCH_SERVICE_NAME` is the name of an Elasticsearch service instance from which the latest backup will be picked
+
+For example:
+
+```
+cf create-service opensearch small-ha-1 my-new-opensearch -c '{"restore_from_latest_backup_of": "my-old-elasticsearch"}'
+```
+
+When you create an OpenSearch backing service in this way, existing data will be available in the OpenSearch service. However new data added to the Elasticsearch service will not be replicated to the OpenSearch service, and vice versa.
```

Comment:
One caveat for restoring from backups: backups are created [hourly on Aiven's unchangeable schedule](https://help.aiven.io/en/articles/4197366-elasticsearch-backups). This means that you kinda have to accept possibly up to one hour of data not being transferred over. This could then be manually migrated, though.

Reply:
Good catch. Will update now.